src/app/admin/report/[term]/new/_components/ParticipantSelect.tsx

```
@@ -0,0 +1,71 @@
+        value={keyword}
+        onChange={(e) => handleKeywordChange(e.target.value)}
+        className="px-4 py-2 outline-none"
+        onFocus={handleFocus}
+        onBlur={handleBlur}
+      />
+      <div
+        className={`${
+          !isEditing && `hidden`
+        } fixed z-10 mt-2 h-fit max-h-[50vh] w-56 overflow-scroll rounded-md border border-gray-100 bg-white shadow-lg`}
+      >
+        <div tabIndex={0} onFocus={handleFocus} onBlur={handleBlur}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)